src/components/CommentList/CommentList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropType from 'prop-types';
+import { commentType } from '../../types';
+import { Comment } from '../Comment';
+
+export const CommentList = ({ comments }) => (
+  comments.map(
+    comment => (
+      <div key={comment.id}>
+        <Comment {...comment} />
+      </div>
+    ),
+  )
+);
```

Comment:
let's add wrapper for a comment list, if you need wrapper for a single comment move it into the Comment component

```suggestion
export const CommentList = ({ comments }) => (
  <div>
  {comments.map(
    comment => (
        <Comment {...comment} key={comment.id} />
    ),
  )}
   </div>
);
```